storage/Devices/MdImpl.cc

```diff
@@ -371,6 +372,15 @@ namespace storage
     }
 
 
+    static
+    bool file_exists(const string& pathname)
```

Comment:
we should probably use `boost::filesystem::exists`
see also `checkDir` and `checkNormalFile` in AppUtil.cc

Reply:
I would wait until we can use C++17 where filesystem is included.